### PR TITLE
add time skew problem to troubleshooting guide

### PR DESCRIPTION
**Pull request by frankbu**

### CODE REVIEW

**Comment by ijsnellf** [comment]
Mention this is a Docker for Mac issue?

**Comment by ijsnellf** [comment]
Never mind, you mention it in the section title.

File: troubleshooting/index.md [hunk 1]
```diff
@@ -10,6 +10,26 @@ type: markdown
...
+You can also confirm this problem by comparing the date inside a docker container to outside:
+
+```bash
+$ docker run --entrypoint date gcr.io/istio-testing/ubuntu-16-04-slave:latest
+Sun Jun 11 11:44:18 UTC 2017
+$ date -u
+Thu Jun 15 02:25:42 UTC 2017
+```
+
+To fix the problem, you'll need to shutdown and then restart Docker before reinstalling Istio.
```
Comment (mccv) [inline troubleshooting/index.md]: If you want to drop a CLI solution in here -

`osascript -e 'quit app "Docker"' && open -a Docker`

Comment (frankbu) [inline troubleshooting/index.md]: Probably not needed, since people should know how to restart (I'd just hit the restart button on the little whale icon)

**Pull request merged**